install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md

```diff
@@ -168,7 +168,7 @@ In your environment, run the following command from your terminal to install Kub
 helm upgrade -i kubecost \
 oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
 --namespace kubecost --create-namespace \
--f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-eks-cost-monitoring.yaml
+-f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/eks-v2.2/cost-analyzer/values-eks-cost-monitoring.yaml
```

Comment:
This isn't a valid branch name. Don't hard-code anything in here, just use something like `<$VERSION>` like above.

Reply:
GitHub allowed me to create a branch named this, and the link seems to work, but I can just create a new one `2.2` or something like that, is that better?

Reply:
Wait, why are you creating new branches in the Helm chart repository? You don't need to do that, just use the variable so the users know they need to complete it on their end.